x-pack/plugins/actions/server/builtin_action_types/alert_history_es_index/create_alert_history_index_template.ts

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ElasticsearchClient, Logger } from 'src/core/server';
+import { ALERT_HISTORY_PREFIX } from '../../../common';
+import mappings from './mappings.json';
+
+export function getAlertHistoryIndexTemplate() {
+  return {
```

Comment:
I happened to think, it would be useful to "self-document" any resources we're creating, perhaps with a `_meta` field: https://www.elastic.co/guide/en/elasticsearch/reference/current/indices-put-template.html#put-index-template-api-request-body

You know someone's going to be wondering what these are, hopefully such fields end up being visible in various diagnostic tools.

Reply:
Updated in https://github.com/elastic/kibana/pull/94909/commits/19f36735275ccfa290cfdcd65d193cbba85841e8